.github/workflows/scripts/build_pr.sh

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/env bash
+
+export MY_ORG=$(echo "${GITHUB_REPOSITORY}" | awk -F / '{print $1}')
+if [ "${GITHUB_BASE_REF}" != "master" ]; then
+    export SUFFIX="-""${GITHUB_BASE_REF}"
+else
+    export SUFFIX=""
+fi
+
```

Comment:
This is the only real innovation in this script; rather than hard-coding a name, such as `narrative-truss` or `narrative-develop`, get it from the GitHub base branch.